instrument-starters/opentracing-spring-cloud-feign-starter/src/test/java/io/opentracing/contrib/spring/cloud/feign/FeignDefinedUrlTest.java

```diff
@@ -49,7 +50,7 @@
 
   }
 
-  @FeignClient(value = "localService", url = "localhost:13598")
+  @FeignClient(value = "nottraced", url = "localhost:13598")
```

Comment:
Is this just a refactor or was it neede?

Reply:
@pavolloffay You can't have several @FeignClient with the same name anymore, otherwise you get:

```
Caused by: org.springframework.beans.factory.support.BeanDefinitionOverrideException: Invalid bean definition with name 'localService.FeignClientSpecification' defined in null: Cannot register bean definition [Generic bean: class [org.springframework.cloud.openfeign.FeignClientSpecification]; scope=; abstract=false; lazyInit=false; autowireMode=0; dependencyCheck=0; autowireCandidate=true; primary=false; factoryBeanName=null; factoryMethodName=null; initMethodName=null; destroyMethodName=null] for bean 'localService.FeignClientSpecification': There is already[Generic bean: class [org.springframework.cloud.openfeign.FeignClientSpecification]; scope=; abstract=false; lazyInit=false; autowireMode=0; dependencyCheck=0; autowireCandidate=true; primary=false; factoryBeanName=null; factoryMethodName=null; initMethodName=null; destroyMethodName=null] bound.
```

Reply:
Regarding the versioning, I've updated the README file to add Greenwhich. I did not test the (in)compatibility with 2.0 Any suggestion on how to test it?

Reply:
You can create a make goal which would set the SB and SB could version. We were doing that before https://github.com/opentracing-contrib/java-spring-cloud/blob/release-0.1.13/Makefile#L10 and then run it as travis job.

Reply:
Ok, see my last commit and last build. It seems ok.

Reply:
sweet it looks awesome